bcos-executor/src/precompiled/common/ContractShardUtils.cpp

```diff
@@ -91,20 +108,30 @@ std::string ContractShardUtils::getContractShard(
     {
         if (entry)
         {
-            tableName = removePrefix(entry->getField(0), INHERENT_PREFIX);
+            tableName = entry->getField(0);
+            tableName.remove_prefix(INHERENT_PREFIX.length());
         }
         entry = getShard(storage, tableName);
 
     } while (isInherent(entry));
 
     if (entry)
     {
-        return std::string(removePrefix(entry->getField(0), SHARD_ROOT_PREFIX));
+        auto shardName = entry->getField(0);
+        shardName.remove_prefix(SHARD_ROOT_PREFIX.length());
+        return std::string(shardName);
     }
 
     return {};
 }
 
+std::string ContractShardUtils::getContractShard(
```

Comment:
almost same input and same output, same function name, ,why wrap a function again